paramak/parametric_components/hollow_cube.py

```diff
@@ -0,0 +1,78 @@
+
+import cadquery as cq
+from paramak import RotateStraightShape
+from paramak.utils import get_hash
+
+
+class HollowCube(RotateStraightShape):
```

Comment:
As far as I understood, this doesn't use any of the methods of `RotateStraightShape`. Shouldn't it simply inherit from `Shape` ?